lib/rosette/tms/smartling-tms/smartling_file.rb

```diff
@@ -133,6 +145,20 @@ def serializer_const
           @serializer_const ||= Rosette::Core::SerializerId.resolve(serializer_id)
         end
 
+        def commit_log
+          @commit_log ||= rosette_config.datastore.lookup_commit_log(
+            repo_config.name, commit_id
+          )
+        end
+
+        def granularity
+          configurator.phrase_storage_granularity
+        end
+
+        def rev_commit
+          @rev_commit ||= repo_config.repo.get_rev_commit(commit_id)
```

Comment:
Hmm... not sure I understand. This line of code is simply looking up a jGit [`RevCommit`](http://download.eclipse.org/jgit/docs/jgit-2.0.0.201206130900-r/apidocs/org/eclipse/jgit/revwalk/RevCommit.html) object for the given commit id.